content/software/ide-v1/tutorials/ide-v1-security/ide-v1-security.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 'Security of Arduino IDE'
+description: 'Learn the secure development process behind Arduino IDE.'
+tags:
+ - Security
+author: 'Arduino Security Team'
+---
+
+Arduino IDE 2.x is the latest version of the Arduino Programming tool and it is built on Eclipse Theia, an open-source framework for building IDEs.

```

Comment:
This needs to change to IDE 1.8.x (or legacy IDE).

Reply:
I know this page is under the IDE v1, however the content of what comes right afterwards refers to the IDE v2. The main purpose for this article for being under the IDE v1 I think is within section "Version Management".

Reply:
Federico is correct, the sentence is and remains about IDE2 even if the article is inside IDE1 page. We want IDE1 users to understand that they need to move to IDE2 which is the latest and maintained. see the story on version management in this page